core/android/common/src/main/kotlin/app/k9mail/core/android/common/database/CursorExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package app.k9mail.core.android.common.database
```

Comment:
We already have a `CursorExtensions.kt` file. Maybe these should be combined?

Reply:
I'll have a look

Reply:
I joined them